addons/remix-tree-button/addon.json

```diff
@@ -21,7 +25,30 @@
       "matches": ["projects"]
     }
   ],
+  "settings": [
+    {
+      "name": "Link placement",
+      "id": "type",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "replace",
+          "name": "Replace \"Remixes \u2192 View all\" link"
+        },
+        {
+          "id": "new",
+          "name": "Add button next to \"Copy Link\""
+        }
+      ],
+      "default": "new"
+    }
+  ],
   "dynamicEnable": true,
   "dynamicDisable": true,
-  "versionAdded": "1.2.0"
+  "versionAdded": "1.2.0",
+  "latestUpdate": {
+    "version": "1.33.0",
+    "temporaryNotice": "The link to the remix tree has moved. If you'd like to go back to the old behavior, you can change the setting below.",
```

Comment:
Not sure if it's a good idea to force this... I bet many users won't find the button

Reply:
Unless we do something like this: https://github.com/ScratchAddons/ScratchAddons/pull/3603#issuecomment-961820896

Reply:
We could also try changing the default but only for users who have this addon disabled at the moment of the update (and for new users too, of course).

Reply:
> Not sure if it's a good idea to force this... I bet many users won't find the button
>
> We could also try changing the default but only for users who have this addon disabled at the moment of the update (and for new users too, of course).

I was definitely thinking about that.

Reply:
Turns out, I accidentally set the default to "new" instead of "replace", so the default hasn't changed, but I'm currently reconsidering what the default should be anyway.

Reply:
The default is now "new".